src/main/java/de/chojo/repbot/listener/StateListener.java

```diff
@@ -62,46 +50,43 @@ public void onGuildJoin(@NotNull GuildJoinEvent event) {
             }
         }
 
-        guildData.migrated(event.getGuild());
+        guilds.guild(event.getGuild()).migration().migrated();
     }
 
     @Override
     public void onGuildLeave(@NotNull GuildLeaveEvent event) {
-        if (!configuration.migration().isActive()) {
-            gdprData.queueGuildDeletion(event.getGuild());
-        }
+        if (!configuration.migration().isActive()) return;
```

Comment:
Fix condition
```suggestion
        if (configuration.migration().isActive()) return;
```
or like the screenshot
```suggestion
```

Reply:
I fixed it. I will also create an issue to remove the migration implementation after a grace period for not migrated guilds